programs/disks/DisksClient.cpp

```diff
@@ -53,34 +62,43 @@ std::vector<String> DiskWithPath::listAllFilesByPath(const String & any_path) co
     return {};
 }
 
-std::vector<String> DiskWithPath::getAllFilesByPattern(const String & pattern) const
+std::vector<String> DiskWithPath::getAllFilesByPattern(const String & pattern, bool ignore_exception) const
```

Comment:
```suggestion
std::vector<String> DiskWithPath::getAllFilesByPrefix(const String & filename_prefix, bool ignore_exception) const
```
I think that would make it more clear about what the code does, because pattern usually means regexp